### ISSUE TITLE
Food with bowls in icon should require a wooden bowl

### ISSUE THREAD

**Issue by lawremi** [iguanaman/HungerOverhaul#11]
Foods like salads should require a wooden bowl in the same way as mushroom soup. This puts constraints on the amount of food stored and should benefit gameplay. For example, fruit salad could have a wooden bowl in place of the mixing bowl.

**Comment by iguanaman** [comment]
I agree, the only problem is the reluctance to modify HarvestCraft recipes I mentioned in the other issue.  Perhaps something that should be petitioned to be in base HC.  I certainly wouldn't object if you wanted to create a pull request though.  

Unfortunately, I've not had time to look into the balance since Pam changed many recipes in the 1.6 move.  I believe HC used to use wooden bowls and the mixing bowl was introduced fairly recently.  Also, there's the issue with 1.7 Forge coming reasonable soon, and I'm kind of holding off to see what changes.
